libcontainer/cgroups/utils.go

```diff
@@ -24,6 +24,10 @@ import (
 const (
 	CgroupProcesses   = "cgroup.procs"
 	unifiedMountpoint = "/sys/fs/cgroup"
+
+	// maxShares corresponds with MAX_SHARES in the Linux kernel
```

Comment:
`s/with/to/`?

Also missing period at the end of the sentense.

Reply:
> `s/with/to/`?

Good question, I alway use "with", but "to" is also possible; https://english.stackexchange.com/questions/142495/correspond-to-vs-correspond-with

I'm fine with either, so let me update